Fix/swap tip

Kahnchan opened the pull request:
<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

* **New Features**
  * Swap Tips now display as interactive tiles; tap the tile to open details.
  * Optional icon and description fields are shown when available.
  * Close button lets you dismiss tips; your preference is persisted.

* **Improvements**
  * Responsive layout: separate title and description on larger screens; concise combined line on smaller screens.
  * Enhanced visuals with borders, hover/press states, shadows, and focus styling.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

Kahnchan committed:
feat: enhance SwapTipsContainer with improved UI and new properties

- Refactored SwapTipsContainer to utilize a more responsive layout with YStack and XStack components.
- Added new properties to ISwapTips interface: iconImage and description for enhanced tip display.
- Implemented close functionality for user-initiated tip dismissal, improving user experience.
- Updated component styling to align with design standards and ensure better accessibility.

Kahnchan committed:
feat: enhance SwapTipsContainer with platform-specific styling

- Added platform-specific shadow and elevation styles for native, Android, and web platforms to improve visual consistency and user experience.
- Updated component styling to align with design standards and enhance accessibility.

revan-zhang commented:
### :tada: **Snyk checks have passed. No issues have been found so far.**

:white_check_mark: **security/snyk** check is complete. No issues have been found. [(View Details)](https://app.snyk.io/org/rayston92/pr-checks/4a14e73d-518c-4678-a4ea-eb9a40774651?refs=f39aebb2a5b246292b994c222b6451b44916b485&source=prChecksComment)

:white_check_mark: **license/snyk** check is complete. No issues have been found. [(View Details)](https://app.snyk.io/org/rayston92/pr-checks/4a14e73d-518c-4678-a4ea-eb9a40774651/license?refs=f39aebb2a5b246292b994c222b6451b44916b485&source=prChecksComment)

originalix commented on packages/kit/src/views/Swap/pages/components/SwapTipsContainer.tsx:
```diff
@@ -1,39 +1,150 @@
-  const intl = useIntl();
+  const [swapTips, setSwapTips] = useSwapTipsAtom();
+  const { gtSm } = useMedia();
+  const themeVariant = useThemeVariant();
```
这个变量没有被使用，删了吧

Kahnchan replied:
okkkk

Kahnchan committed:
Merge branch 'x' into fix/swap_tip

Kahnchan committed:
refactor: simplify SwapTipsContainer by removing unused imports

- Removed the unused `useThemeVariant` import to clean up the code and improve readability.
- Minor adjustments to maintain component functionality.

Kahnchan committed:
Merge branch 'x' into fix/swap_tip
